lib/ssl/doc/guides/using_ssl.md

```diff
@@ -875,7 +875,7 @@ two mechanisms. Bloom filters are fast, memory-efficient, probabilistic data
 structures that can tell if an element may be in a set or if it is definitely
 not in the set.
 
-If the option [anti_replay](`t:ssl:anti_replay/0`) is defined in the server, a
+If the option [anti_replay] is defined in the server, a
```

Comment:
```suggestion
If the option `t:anti_replay/0` is defined in the server, a
```

Reply:
I have not keept this type

Reply:
Then you probably want to change it to `` `anti_replay` `` instead of `[anti_replay]`.

Reply:
fixed